docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.4"
+
+services:
+  db:
+    image: postgres:11.5-alpine
+    tmpfs: /var/lib/postgresql/data
+    ports:
+      - "${PORT_DB-5432}:5432"
+    environment:
+      POSTGRES_DB: graphile_worker_test
+    container_name: graphile_worker_db
```

Comment:
If this just used the `db` container_name would it conflict with other things on the system, or is the container_name local to this docker-compose somehow? (Docker noob here.)

Reply:
It needs to be globally unique if we want to execute commands in the container using docker (bypassing compose - see https://github.com/gregplaysguitar/worker/blob/55a7926731b3277fc57ce62e4c9b82109825d7ef/README.md#L618). 

The reason for that was that I was getting "the input device is not a TTY" when trying to run it via compose... however I just worked out that passing `-T` fixes that so I'll update. Definitely better to keep it local.

Reply:
:+1: Thanks for explanation